pyannote/audio/applications/multilabel.py

```diff
@@ -0,0 +1,564 @@
+  --database=<database.yml>        Path to database configuration file.
+  --subset=<subset>          Set subset (train|developement|test).
+                             Defaults to "train" in "train" mode. Defaults to
+                             "development" in "validate" mode. Defaults to all subsets in
+                             "apply" mode.
+  --gpu                      Run on GPUs. Defaults to using CPUs.
+  --batch=<size>             Set batch size. Has no effect in "train" mode.
+                             [default:  32]
+  --from=<epoch>             Start {train|validat}ing at epoch <epoch>. Has no
+                             effect in "apply" mode. [default:  0]
+  --to=<epochs>              End {train|validat}ing at epoch <epoch>.
+                             Defaults to keep going forever.
+"train" mode: 
+  <experiment_dir>           Set experiment root directory. This script expects
+                             a configuration file called "config.yml" to live
+                             in this directory. See "Configuration file"
+                             section below for more details.
+
+"validation" mode: 
+  --every=<epoch>            Validate model every <epoch> epochs [default:  1].
+  --chronological            Force validation in chronological order.
+  --parallel=<n_jobs>        Process <n_jobs> files in parallel. Defaults to
+                             using all CPUs.
+  <label>                    Label to predict (KCHI, CHI, FEM, MAL or speech).
+                             If options overlap and speech have been activated during training, one
+                             can also to validate on the classes (SPEECH, OVERLAP).
+  <train_dir>                Path to the directory containing pre-trained
+                             models (i.e. the output of "train" mode).
+  --precision=<precision>    Target detection precision [default:  0.8].
+  --use_der                  Indicates if the Detection Error Rate should be used for validating the mode.
+                             Default mode uses precision/recall.
+
+"apply" mode: 
+  <model.pt>                 Path to the pretrained model.
+  --step=<step>              Sliding window step, in seconds.
+                             Defaults to 25% of window duration.
+
+Database configuration file <database.yml>: 
+    The database configuration provides details as to where actual files are
+    stored. See `pyannote.database.util.FileFinder` docstring for more
+    information on the expected format.
+
+Configuration file: 
+    The configuration of each experiment is described in a file called
+    <experiment_dir>/config.yml, that describes the feature extraction process,
+    the neural network architecture, and the task addressed.
+
+    ................... <experiment_dir>/config.yml ...................
+    # train the network for segmentation
+    # see pyannote.audio.labeling.tasks for more details
+    task: 
+       name:  Segmentation
```

Comment:
Can you please make a new pass on the sample configuration file?
I believe it has not been updated for the multi-label thing...